CONTRIBUTING.md

```diff
@@ -0,0 +1,188 @@
+# Contributing to JCV
+
+The following is a set of guidelines for contributing to JCV, which are hosted in the Ekino Organization on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Java Styleguide](#java-styleguide)
+  * [Kotlin Styleguide](#kotlin-styleguide)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for Atom. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
```

Comment:
`for Atom` I think JCV would be better here :)

Reply:
oupss ...my bad ...